build.gradle

```diff
@@ -30,6 +30,8 @@ repositories {
 
 dependencies {
 	compile(group: 'bio.terra', name: 'stairway', version: '0.0.4.1-SNAPSHOT')
+	compile(group: 'bio.terra', name: 'datarepo-client', version: '1.0.0-SNAPSHOT')
+	compile(group: 'org.glassfish.jersey.inject', name: 'jersey-hk2', version: '2.29.1')
```

Comment:
i pulled this in because without it i was getting `java.lang.IllegalStateException: InjectionManagerFactory not found.` every time the DR client was called. not sure if there was a dependency collision (w/ the Sam client?) or if it was just missing entirely but this fixed it.

Reply:
hrm this is a little troubling, I'd prefer not to clutter our build.gradle if we don't have to - the provenance gets lost. At minimum, add a comment to explain why the dependency is here. At best - contact the data repo folks and ask if the published artifact is missing a dependency?

Reply:
@ddietterich could you please confirm/refute my understanding of this issue?

Reply:
Interesting. Let me do a little digging. I'm not sure why swagger-codegen needs jersey anything to generate the **client** code.

Reply:
Yes, there was a missing dependency in the datarepo-client. I have fixed this and tested it with a jadecli build that doesn't have the jersey-hk2 thing.

Try client 1.0.1-SNAPSHOT and let me know if you can successfully remove that dependency.

Reply:
That fixed it. Thanks Dan!